controllers/upgrade_handlers.go

```diff
@@ -76,5 +172,167 @@ func (r *ImageBasedUpgradeReconciler) handleUpgrade(ctx context.Context, ibu *lc
 		metav1.ConditionFalse,
 		"Upgrade completed",
 		ibu.Generation)
+
 	return doNotRequeue(), nil
 }
+
+func checkIfBackupRequested(restore *backuprestore.BRHandler, content []lcav1alpha1.ConfigMapRef, ctx context.Context) (bool, []*velerov1.Backup, error) {
```

Comment:
Does it make more sense to have these functions as part of the backup plugin code?

Reply:
Yeah! Moved to backup.go